content/en/blog/_posts/2022-07-27-sig-apps-GA-1.25/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: blog
+title: "sig-apps features graduating to stable in 1.25"
+date: 2022-07-27
+slug: "sig-apps features graduating to stable in 1.25"
+---
+
+**Authors:** Ravi Gudimetla (Apple), Filip Krepensky (Red Hat), Maciej Szulik (Red Hat)
+
+This blog describes the two features namely `minReadySeconds for StatefulSets` and `maxSurge for Daemonsets` that sig-apps is happy to graduate to stable in 1.25
+
+## What problems does these features solve?
+
+### MinReadySeconds for StatefulSets
+`minReadySeconds` ensures that the statefulset workload is `Ready` for the given number of seconds before calling the
+pod `Available`. The notion of being `Ready` and `Available` is quiet important for workloads. For example, some workloads like Prometheus with multiple instances of Alertmanager should be considered `Available` only when the Alertmanager's state transfer is complete. `minReadySeconds` also helps when using loadbalancers with cloud providers. Since the pod should be `Ready` for the given number of seconds, it provides buffer time to prevent killing pods in rotation before new pods show up.
```

Comment:
```suggestion
pod `Available`. The notion of being `Ready` and `Available` is quite important for workloads. For example, some workloads like Prometheus with multiple instances of Alertmanager should be considered `Available` only when the Alertmanager's state transfer is complete. `minReadySeconds` also helps when using loadbalancers with cloud providers. Since the pod should be `Ready` for the given number of seconds, it provides buffer time to prevent killing pods in rotation before new pods show up.
```

Reply:
I know the last sentence says it, but maybe we should mention specifically rollouts? We discussed something similar in the docs PR: https://github.com/kubernetes/website/pull/35539#discussion_r933175664